cardano-lib/generic-tracer-config.nix

```diff
@@ -0,0 +1,53 @@
+{
+  # Cardano-tracer will use default values where parameters are set to null.
+  WarnRTViewMissing = null;
+  ekgRequestFreq = null;
```

Comment:
Maybe add comment to clarify:
`# Interval to request metrics forwarding (seconds)`